### PR TITLE
Added lines to populate petZoningInfo to allow writing pet info into …

### PR DESCRIPTION
**Pull request by sleepingkirby**
…char_stats table to retain pets on log out and login.

### CODE REVIEW

File: src/map/utils/petutils.cpp [hunk 1]
```diff
@@ -1096,8 +1096,10 @@ namespace petutils
 
         Pet_t* PPetData = g_PPetList.at(PetID);
 
-        if (PMaster->objtype == TYPE_PC)
+        if (PMaster->objtype == TYPE_PC){
```
Comment (TeoTwawki) [inline src/map/utils/petutils.cpp]: We're braces on newlines folk 'round here pardner 😃

Comment (sleepingkirby) [inline src/map/utils/petutils.cpp]: Will make that change, but it turns out that it works for a single logout/login then something else goes and resets the pet info to zero. Might as well cancel this request until I figure that out >.<.

Comment (TeoTwawki) [inline src/map/utils/petutils.cpp]: Its ok to leave open while you work on it and if you figure it out just updating your branch will automatic update the pull request. *Sometimes* we'll close out really old ones we think got abandoned.

Comment (TeoTwawki) [inline src/map/utils/petutils.cpp]: logout should pretty much be the same as zoning out so surprised this isn't already a thing honestly.

Comment (sleepingkirby) [inline src/map/utils/petutils.cpp]: Branch updated. ^^/

Comment (TeoTwawki) [inline src/map/utils/petutils.cpp]: All set and tested now?

Comment (sleepingkirby) [inline src/map/utils/petutils.cpp]: Yep, Should be all good. My wyvern hasn't disappeared on me while zoning or logging out yet. ^^b

**Pull request merged**